hal/src/main/native/systemcore/AddressableLED.cpp

```diff
@@ -148,8 +178,8 @@ void HAL_SetAddressableLEDData(int32_t start, int32_t length,
     *status = PARAMETER_OUT_OF_RANGE;
     return;
   }
-  // TODO: handle color order
-  std::memcpy(&leds->s_buffer[start * 3], data, length * 3);
+  ConvertAndCopyLEDData(&leds->s_buffer[start * 3], data, length * 3,
```

Comment:
```suggestion
  ConvertAndCopyLEDData(&leds->s_buffer[start * sizeof(HAL_AddressableLEDData)], data, length,
```
Length is the number of pixels.  It appears that ConvertAndCopyLEDData() takes number of pixels, not number of bytes.

While we're here, change the `start * 3` to use sizeof.